templates/tetra-requirements.adoc.liquid

```diff
@@ -0,0 +1,65 @@
+= {{architecture.package.description}}
+:doc-name: tetra-requirements
+:doc-name-adoc: tetra-requirements.adoc
+:revnumber: v{{architecture.package.version}}
+:toc: left
+
+ifndef::backend-pdf[]
+image:https://img.shields.io/badge/License-MIT-yellow.svg[MIT License, link=https://opensource.org/licenses/MIT]
+image:https://img.shields.io/badge/Contribute%20on-GitHub-orange[Contribute on GitHub, link={{architecture.package.repository}}]
+image:https://img.shields.io/badge/Download%20-PDF-blue[Download PDF, link=../dist/{doc-name}.pdf]
+endif::[]
+
+<<<
```

Comment:
This '<<<' is showing up as a weird highlight in git, everything's gonna be OK with this, right?

Reply:
I'll have to poke as to why it's red, but this is just asciidoc syntax for a page break
https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#breaks